src/mesh/region.cc

```diff
@@ -126,62 +95,73 @@ Region RegionDB::add_region( unsigned int id, const std::string &label, unsigned
         THROW(ExcNonuniqueLabel() << EI_Label(label) << EI_ID(id) << EI_IDOfOtherLabel(it_label->get_id()) );
```

Comment:
Check that these exceptions are caught and input address is provided in calling functions. Add into comment to that functions, that they may throw and address has to be provided.